packages/fuels-test-helpers/src/node.rs

```diff
@@ -26,6 +27,8 @@ use tokio::{process::Command, sync::oneshot};
 
 use crate::utils::{into_coin_configs, into_message_configs};
 
+pub const DEFAULT_CACHE_SIZE: usize = 10 * 1024 * 1024;
```

Comment:
What's the unit of this cache size, and where does this default come from?

Reply:
It comes from fuel-core

Reply:
What I mean is that : 
- If this default comes from `fuel-core` then we should use the constant from the `fuel-core` repository. If it's not possible, a comment should explain where this default comes from and why it's there.

Reply:
The explanation will suffice. We can't import anything from fuel-core because of fuel-core.bin.